build/action.yml

```diff
@@ -80,8 +80,18 @@ runs:
         name: build-artifacts
         path: |
           target/*
-          **/target/*.jar
+          ./*/target/*
           **/*source-release.zip
+          !${{ inputs.tests_module_path }}*
+        retention-days: ${{ inputs.github_artifact_retention }}
+
+    - name: Archive test artifacts
+      if: always()
+      uses: actions/upload-artifact@v3
+      with:
+        name: build-artifacts
```

Comment:
you should use a different name

Reply:
The goal is to have all artefact within the same folder (as we want to deploy them in the test image as we do for build module in PR code change)
See https://github.com/actions/upload-artifact#uploading-to-the-same-artifact